incubator/jaeger/README.md

```diff
@@ -48,6 +48,38 @@ helm install incubator/jaeger --name myrel --set cassandra.config.max_heap_size=
 
 > **Tip**: List all releases using `helm list`
 
+## Installing the Chart using an Existing Cassandra Cluster
```

Comment:
Looks like I can't comment on line 7, so, I'll comment here: the readme introduction might need an update, as it says that `This chart also depends on the cassandra chart`, which might not be accurate anymore.

Reply:
oops. Will fix that.